README.md

```
@@ -183,6 +188,24 @@ Examples:
 | Elevation | `low-angle shot` (-30°), `eye-level shot` (0°), `elevated shot` (30°), `high-angle shot` (60°) |
 | Distance | `close-up`, `medium shot`, `wide shot` |
 
+#### JoyAI Image
+
+`JoyAI Image` uses the official [JoyAI-Image camera-control prompt format](https://github.com/jd-opensource/JoyAI-Image#33-camera-control):
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)